Playground for the new commondata format.ipynb

```diff
@@ -483,20 +505,198 @@
     "data_df"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": 61,
+   "id": "41394702",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dataclasses import dataclass\n",
+    "from pathlib import Path\n",
+    "import typing\n",
+    "\n",
+    "from validobj.custom import Parser\n",
+    "from validobj import ValidationError, parse_input\n",
+    "\n",
+    "\n",
+    "from validphys import convolution\n",
+    "\n",
+    "\n",
+    "# Scalar parsers\n",
+    "@Parser\n",
+    "def ValidPath(path_str: str) -> Path:\n",
+    "    \"\"\"Parse strings into paths\"\"\"\n",
+    "    try:\n",
+    "        return Path(path_str)\n",
+    "    except exception as e:\n",
+    "        raise ValidationError(f\"{path_str} is not a valid path\") from e\n",
+    "        \n",
+    "@Parser\n",
+    "def ValidOperation(op_str: str) -> str:\n",
```

Comment:
Would it make sense to use enums, or Union[Literal[str]] for better structure here?

Reply:
For the OPs list you mean?

Reply:
Yes indeed. Would make for better error messages.